src/main/java/Ui.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Scanner;
+
+public class Ui {
+    public void initiate(Storage storage, TaskList taskList) {
+        String logo = " ____        _        \n"
```

Comment:
This method's functionality should probably belong to the `Duke` class instead? Seems that the main loop for the program is now placed here, which isn't what we typically expect a UI component to do -- to only handle inputs/outputs.